Gemfile

```diff
@@ -3,10 +3,19 @@ source "https://rubygems.org"
 gemspec
 
 gem 'gettext', '>= 3.1.3', '< 4.0.0'
-gem 'rake', '~> 13.1.0'
+
+group :test do
+  gem 'ci_reporter', '>= 1.6.3', '< 2.0.0', :require => false
+  gem 'minitest'
+  gem 'minitest-spec-context'
+  gem 'mocha'
+  gem 'rake', '~> 13.0'
```

Comment:
I don't think we need to pin rake. If tests are broken, we should fix that. If there's a breaking change in the lib, we should adjust.